### ISSUE TITLE
error google.protobuf.Empty when destroying mongodb cluster

### ISSUE THREAD

**Issue by juwit** [yandex-cloud/terraform-provider-yandex#35]
Hi there,

I encountered the error `message type "google.protobuf.Empty" isn't linked in
` when creating/destroying mongodb clusters using the `yandex_mdb_mongodb_cluster` resource.
The cluster is successfully deleted from Yandex Cloud, but the terraform state isn't updated as the command fails.

Here is an extract of my terraform resource :
```hcl
data "yandex_vpc_network" "network" {
  name = "network"
}

data "yandex_vpc_subnet" "persistence" {
  name = "development-persistence-private-ru-central1-a"
}

resource "yandex_mdb_mongodb_cluster" "this" {
  name        = "development-cluster"
  environment = "PRESTABLE"
  network_id  = data.yandex_vpc_network.network.id

  cluster_config {
    version = "4.2"
  }

  labels = {
    environment = "development"
  }

  database {
    name = "development-cluster"
  }

  user {
    name     = "mongod"
    password = "password"
    permission {
      database_name = "development-cluster"
    }
  }

  resources {
    resource_preset_id = "b2.nano"
    disk_size          = 16
    disk_type_id       = "network-hdd"
  }

  host {
    zone_id   = "ru-central1-a"
    subnet_id = data.yandex_vpc_subnet.persistence.id
  }
}
```

Here is a DEBUG log from the command `TF_LOG=DEBUG terraform destroy` :
```log
2019-12-31T15:36:55.758+0100 [INFO]  plugin.terraform-provider-yandex_v0.27.0_x4: configuring server automatic mTLS: timestamp=2019-12-31T15:36:55.758+0100
2019-12-31T15:36:55.788+0100 [DEBUG] plugin: using plugin: version=5
2019-12-31T15:36:55.788+0100 [DEBUG] plugin.terraform-provider-yandex_v0.27.0_x4: plugin address: network=unix address=/tmp/plugin983585600 timestamp=2019-12-31T15:36:55.788+0100
2019/12/31 15:36:55 [DEBUG] yandex_mdb_mongodb_cluster.this: applying the planned Delete change
2019/12/31 15:36:55 [TRACE] GRPCProvider: ApplyResourceChange
yandex_mdb_mongodb_cluster.this: Destroying... [id=c9q2tjdr541ls6vmscvk]
2019-12-31T15:36:55.864+0100 [DEBUG] plugin.terraform-provider-yandex_v0.27.0_x4: 2019/12/31 15:36:55 [DEBUG] Deleting Mongodb Cluster "c9q2tjdr541ls6vmscvk"
yandex_mdb_mongodb_cluster.this: Still destroying... [id=c9q2tjdr541ls6vmscvk, 10s elapsed]
2019/12/31 15:37:10 [DEBUG] yandex_mdb_mongodb_cluster.this: apply errored, but we're indicating that via the Error pointer rather than returning it: any: message type "google.protobuf.Empty" isn't linked in
2019/12/31 15:37:10 [TRACE] <root>: eval: *terraform.EvalWriteState
2019/12/31 15:37:10 [ERROR] <root>: eval: *terraform.EvalApplyPost, err: any: message type "google.protobuf.Empty" isn't linked in
2019/12/31 15:37:10 [ERROR] <root>: eval: *terraform.EvalSequence, err: any: message type "google.protobuf.Empty" isn't linked in
2019/12/31 15:37:10 [ERROR] <root>: eval: *terraform.EvalOpFilter, err: any: message type "google.protobuf.Empty" isn't linked in
2019/12/31 15:37:10 [TRACE] [walkDestroy] Exiting eval tree: yandex_mdb_mongodb_cluster.this (destroy)

Error: any: message type "google.protobuf.Empty" isn't linked in
```

**Comment by GennadySpb** [comment]
Hi @juwit !
Thanks for report! It will be fixed in next release of provider: SDK [commit](https://github.com/yandex-cloud/go-sdk/commit/abfe0e92149fe7e3b31ac31bed17558e0be6f416) with fix already published.

**Comment by GennadySpb** [comment]
Fix in release 0.28.0
